packages/components/src/ConfirmationModal/ConfirmationModal.mdx

```diff
@@ -109,6 +112,32 @@ that presents a confirm modal.
   }}
 </Playground>
 
+## Destructive
+
+Sometimes when the action is critical or destructive in nature (e.g. Deleting
+user or account), you may want to use `destructive type` modal.
```

Comment:
I don't think that we need to give an example here. (deleting user of account)

Reply:
```suggestion
If the user is confirming an action that is will destroy, delete, or remove something, use a destructive ConfirmationModal to visually reinforce the potential consequences.
```